src/main/java/Rolex.java

```diff
@@ -0,0 +1,140 @@
+import tasks.Task;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Todo;
+import exception.RolexException;
+
+import java.util.Scanner;
+
+public class Rolex {
+
+    static Task[] task =  new Task[100];
+    static int taskCount = 0;
+    public static void printLines(){
+        System.out.println("--------------------------------------------------");
+    }
+
+    public static void rolexGreetsUser(){
+        System.out.println("Hello! I'm ROLEX");
+        System.out.println("What can I do for you?\n");
+    }
+
+    public static void inputIsBye(){
+        printLines();
+        System.out.println("Bye. Hope to see you again soon!\n");
+        printLines();
+    }
+
+    public static void inputIsList(){
+        if(taskCount>=1){
+            printLines();
+            int indexNum = 1;
+            System.out.println("Here are the tasks in your list:");
+            for(int i = 0; i<taskCount; i++){
+                System.out.println(indexNum + ". " + task[i]);
+                indexNum++;
+            }
+            printLines();
+        } else{
+            RolexException.detectError("list");
+        }
+    }
+
+    public static void inputIsMark(String userInput){
+        int index = Integer.parseInt(userInput.substring(5));
+        if(index>0 && index<=taskCount) {
+            task[index-1].MarkTask();
+        } else{
+            RolexException.detectError(userInput);
+        }
+    }
+
+    public static void inputIsUnmark(String userInput){
```

Comment:
Try to rephrase your method names to act as verbs rather than boolean conditionals (eg. handleMarkInput)

Reply:
I have seen all the reviews from you and will format accordingly. Thanks @theopin !